config/samples/core_v1beta1_openstackcontrolplane.yaml

```diff
@@ -104,6 +104,53 @@ spec:
   nova:
     template:
       secret: osp-secret
+  heat:
+    enabled: false
+    template:
+      customServiceConfig: "# add your customization here"
+      databaseInstance: openstack
+      databaseUser: "heat"
+      rabbitMqClusterName: rabbitmq
+      debug:
+        bootstrap: false
+        dbSync:
+        service: false
+      heatAPI:
+        containerImage: "quay.io/tripleozedcentos9/openstack-heat-api:current-tripleo"
```

Comment:
It might be better here and in the other samples to remove these `containerImage` fields and instead integrate the Heat defaulting mechanism for container images.  See [1] and [2] for an example.

[1] https://github.com/openstack-k8s-operators/openstack-operator/blob/e20f67d6a480aed5bec97a750fba68682336f1f5/main.go#L294-L299
[2] https://github.com/openstack-k8s-operators/openstack-operator/blob/e20f67d6a480aed5bec97a750fba68682336f1f5/apis/core/v1beta1/openstackcontrolplane_webhook.go#L292-L295

Reply:
Done